.prettierignore

```diff
@@ -0,0 +1,5 @@
+# Ignore artifacts:
+node_modules
+dist
+TODO.md
+readme.md
```

Comment:
use pattern like `*.md` so you don't have to ignore each `md` file separately

Reply:
Added and pushed to the branch.